content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -521,26 +521,24 @@ This configuration is typically found in the `/etc/falco/falco.yaml` file. If Fa
 
 1. Create a [kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) for the [kube-apiserver][kube-apiserver] webhook audit backend.
 
-    ```shell
-    cat <<EOF > /etc/kubernetes/audit-webhook-kubeconfig
-    apiVersion: v1
-    kind: Config
-    clusters:
-    - cluster:
-        server: http://<ip_of_falco>:8765/k8s_audit
-      name: falco
-    contexts:
-    - context:
-        cluster: falco
-        user: ""
-      name: default-context
-    current-context: default-context
-    preferences: {}
-    users: []
-    EOF
-    ```
+        cat <<EOF > /etc/kubernetes/audit-webhook-kubeconfig
```

Comment:
_mini-nit_
This loses the formatting as Bourne-style shell. Ideally: fix the Markdown and retain the code formatting.

Reply:
Just used the same style the logstash conf uses which renders it. For some reason the other styles didn't render so I had to use this one.